indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -282,6 +282,36 @@ $i-box-padding: 10px;
     }
 }
 
+.i-box {
```

Comment:
sass-lint will call you out for that! ;)

I think it'd make sense to combine the two